.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'textwrap'
+        dry-run: false
```

Comment:
The page you linked says that `dry-run: false` is the default value — I would thus prefer to remove this line.

Reply:
Done, thanks!